.testrunner:
  image: ghcr.io/datadog/dd-trace-py/testrunner:8b69a2610342b333f8832422ffc4f3a9327bed13@sha256:c2d067947ffdb305fc7dc7ff1f8eb7035cfa110bd1199917dd2519eadd166402
  # DEV: we have a larger pool of amd64 runners, prefer that over arm64
  tags: [ "arch:amd64" ]
  timeout: 20m
  before_script:
    - ulimit -c unlimited
    - pyenv global 3.12 3.8 3.9 3.10 3.11 3.13
    - export _CI_DD_AGENT_URL=http://${HOST_IP}:8126/
  retry: 2
  artifacts:
    reports:
      junit: test-results/junit*.xml
